contracts/l2/dai.sol

```diff
@@ -38,16 +38,15 @@ contract Dai {
   // --- ERC20 Data ---
   string  public constant name     = "Dai Stablecoin";
   string  public constant symbol   = "DAI";
-  string  public constant version  = "1";
   uint8   public constant decimals = 18;
   uint256 public totalSupply;
 
   mapping (address => uint256)                      public balanceOf;
   mapping (address => mapping (address => uint256)) public allowance;
   mapping (address => uint256)                      public nonces;
 
-  event Approval(address indexed src, address indexed guy, uint256 wad);
```

Comment:
Not sure how the others feel about this, but it doesn't feel like Dai without the Daiwanese.

Reply:
In this case I felt it more natural to stick with the common ERC20 language, but again we can go back to Daiwanese if the rest of the team feels it's a better fit.